fe-nc-news/src/Articles.jsx

```diff
@@ -0,0 +1,38 @@
+import getArticles from "./utils/Articles";
+import React from 'react'
+import {useState, useEffect} from 'react'
+import "./articles.css"
+
+const Articles = () => {
+    const [articles, setArticles] = useState([])
+
```

Comment:
Good work, would like to see some loading state for user feedback for the asynchronous getArticles.